forwarder/update_handlers/base_update_handler.py

```diff
@@ -19,11 +19,16 @@ def __init__(self, producer: KafkaProducer, pv_name: str, output_topic: str):
             year=1900, month=1, day=1, hour=0, minute=0, second=0, tzinfo=timezone.utc
         )
 
-    def _publish_message(self, message: bytes, timestamp_ns: Union[int, float]) -> None:
+    def _publish_message(self, message: Optional[bytes], timestamp_ns: Union[int, float]) -> None:
+        if message is None:
+            self._logger.error(
+                f'Rejecting update from PV "{self._pv_name}" as the message was not serialised.'
+            )
+            return
```

Comment:
if we use type hints, I suppose the correct thing would be to replace
return with return None. For example mypy would complain otherwise I believe.